src/main/java/com/monumental/models/Contribution.java

```diff
@@ -0,0 +1,56 @@
+package com.monumental.models;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Model class for a Contribution
+ * Contains all of the state associated with a Contribution along with Getters and Setters for that state
+ */
```

Comment:
Random thought: she may want:
1. Users to have a public profile to show their contributions
2. Users to have a private page with the same info

Not saying we should promise any of that now, but it makes sense down the road once users exist